### DISCUSSION TITLE
zh-cn/manual/introduction/combat

### DISCUSSION THREAD

**Discussion by giscus** [MaaAssistantArknights/maa-website#141] [Comments]
# zh-cn/manual/introduction/combat

刷理智 常规设置 吃理智药 + 吃源石 和 指定次数、指定材料 三个选项为短路开关（或门），即达成三个选项中的任一条件，均会视为任务完成，停止刷理智。 吃理智药 指定补充几次理智（可能一次吃多瓶药）。 吃源石 指定碎几颗石头（一次一颗），当仓库中有理智药时不会碎石。 指定次数 指定刷多少次指定关卡（例如“刷 15 次后停止”）。 指定材料 指定刷多少个...

https://docs.maa.plus/zh-cn/manual/introduction/combat.html

**Comment by 2407135388** [comment]
是否能让使用者自由添加更多的备用关卡
现有的三个 我的顺序为剿灭》龙门币》1-7
我想在1-7之上再添加红票关

**Reply by su-iras-1** [reply]
建议加入刷取逻辑 
关卡1_选择关卡_刷多少次
关卡2_选择关卡_刷多少次
关卡2_选择关卡_刷多少次
如果没有理智 自动跳过剩下关卡 结束理智作战任务

**Comment by hurew119** [comment]
能否手动延长进入关卡的等待时间？
我的电脑没有GPU，使用模拟器进入关卡加载时间很长（3~5min，加载后能正常战斗）
现在MAA会在进图的等待阶段报刷理智错误，然后自动跳到下一个任务

**Comment by Asymptote-404** [comment]
最近理智代理是不是有问题？选的红票 备选是龙门币 近特芯片。 一直理智出错，今天看了一下 是点进旁边的技能本了

**Comment by Suzuha08** [comment]
感觉理智作战这一块写的不是很好，逻辑有点奇怪

**Comment by su-iras-1** [comment]
提问“周计划”怎么用？

**Reply by ABA2396** [reply]
没勾的那天就会跳过这个任务

**Comment by human-sketch** [comment]
能問一下有沒有人知道如何在linux的客戶端上弄maa嗎，兼容WINE嗎？

**Reply by WuNingErSuo** [reply]
maa-cli可跨平台使用。

**Reply by human-sketch** [reply]
**要怎么弄**

**Comment by Bismooth** [comment]
希望理智作战能支持自定义类似自动战斗那样的关卡队列。
像旧活动复刻这种场景，以前打过一遍，已经有代理了，只需要按顺序重新打一遍就可以领取奖励，不需要抄作业。注意到SSReopen-XXX是这个功能，但是
1. 似乎只支持普通关卡刷一遍（还没有自测EX这种是否支持）
2.SSReopen似乎需要开发者根据当期活动更新资源提供，如果能提供自定义、代理执行队列可以节省这样的人力。

edit：
我想用自动战斗+prts plus上面随便找到的作业集实现自动刷所有关卡领取奖励的效果。但是会**自动取消战斗代理**，即使关闭了自动队列也无法点选代理，所以很遗憾不行。
